src/pages/workspaces/workspace/jobHistory/SubmissionDetails.js

```diff
@@ -150,18 +150,15 @@ const SubmissionDetails = _.flow(
     submissionDetailsBreadcrumbSubtitle(namespace, name, submissionId),
     _.isEmpty(submission) ? centeredSpinner() : h(Fragment, [
       div({ style: { display: 'grid', gridTemplateColumns: '1fr 4fr' } }, [
-        div({ style: { display: 'grid', gridTemplateRows: 'repeat(3, 33%)' } }, [
+        div({ style: { display: 'grid', gridAutoRows: 'min-content'} }, [
```

Comment:
So previously we forced 3 equal sized rows, what happens now? Do we have as many rows as we need, each of which resizes up or down according to content?

Reply:
Yes. I attached screenshots to the MR description.

Reply:
@cjllanwarne [This](https://bitsofco.de/understanding-the-difference-between-grid-template-and-grid-auto) might help clarify the distinction between `grid-template-*` and `grid-auto-*`.